pkg/operator/ceph/controller/spec.go

```diff
@@ -686,15 +690,21 @@ func PrivilegedContext(runAsRoot bool) *v1.SecurityContext {
 func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Container {
 
 	var maxLogSize resource.Quantity
+	rotation := "7"
```

Comment:
nit: move this down to line 703 right next to where it's overridden

Reply:
done